src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LanguageServerProjectSystem.cs

```diff
@@ -98,6 +98,8 @@ public async Task OpenSolutionAsync(string solutionFilePath)
     {
         using (await _gate.DisposableWaitAsync())
         {
+            await ProjectInitializationHandler.SendProjectInitializationStartedNotificationAsync();
```

Comment:
Fine with a notification for now.  But definitely in the future I would like us to move to a generic progress request that is triggered when the project load/reload is triggered - right now the project could be reloading and we won't show a busy indicator.